### ISSUE TITLE
Exception in server tick loop - randomly generated dimension

### ISSUE THREAD

**Issue by Stat1c-V01D** [McJty/RFToolsDimensions#326]
Issue description: 
I have created a random world. Somehow it decided to be a world full of seeds on a ground that those can't be planted on.
Shortly after the whole server started lagging out and soon it crashed.
Now everytime I try to restart it crashes after loading the overworld.
This is on the newest version of the Direwolf20 1.16 Pack btw.

Steps to reproduce: 
No idea because it was a randomly generated world.

Versions:
* Minecraft: 1.16.5
* Forge: 1.16.5-36.0.45
* McJtyLib: 1.16-5.0.17
* RFTools: 1.16-2.0.10
* RFTools Dimensions: 1.16-7.0.4-beta

Possibly incompatible mods: 
none that i know of

Relevant logs, if any: 
https://pastebin.com/je8EkDkj

**Comment by McJty** [comment]
I will check it out

**Comment by Stat1c-V01D** [comment]
I have fumbled around with the world save and data files.
The hacky solution I found for now is to delete the "RFToolsDimensions.dat" together with the corrupted dimension wold folder.
The server grumbles a bit about a missing dimension but spawns the player into the overworld without loosing anything.
Other dimensions that were generated previously seem fine. No deleted items,block or machines even from different mods.
I can not tell how far this will screw things up in the future but for the last couple of random test dimensions everything seems fine for now.
